particle-iot/device-os
# [Gen 3] Updates embedded bootloader

### Problem

The Gen 3 embedded bootloader was supposed to be updated in 1.1.0-rc.1.

### Solution

Update the embedded bootloader for Argon, Boron, Xenon to v301 and include in system-part1 (not SoMs ... flash bootloader manually for those).

This PR also bumps module versions in preparation for v1.1.0-rc.2

**Workaround** flash the 1.1.0-rc.1 bootloader manually via `particle flash --serial bootloader.bin`

### Steps to Test

Using the example app provided in #1752, test that 1.1.0-rc.1 behaves as stated when sleeping and resetting.  Flash this firmware via `modules $ make clean all -s PLATFORM=xenon COMPILE_LTO=n APP=tinker program-dfu` and verify after sleeping and resetting the device does not hard fault anymore.

### References

- Closes #1752 
- [CH31484]
---

### Completeness


---

- [bugfix] [Gen 3] Updates embedded bootloader [#1756](https://github.com/particle-iot/device-os/pull/1756)
